src/stats/targets.ts

```diff
@@ -0,0 +1,76 @@
+import { EventEmitter } from "events";
+import { last } from "lodash";
+
+import type { FrameEntryType, FramesType } from "../types";
+import type { TargetBreakType } from "./common";
+import type { StatComputer } from "./stats";
+
+// The Target item's in-game ID
+const TARGET_ITEM_TYPE_ID = 209;
+
+export class TargetBreakComputer extends EventEmitter implements StatComputer<TargetBreakType[]> {
+  private targetBreaks = new Array<TargetBreakType>();
+
+  public constructor() {
+    super();
+  }
+
+  public setup(): void {
+    // Reset the state
+    this.targetBreaks = [];
+  }
```

Comment:
`setup()` gets a copy of the `GameSettingsType`. We should determine whether or not this current game is a target test game and if not do nothing on `processFrame()`.

E.g. we can do this:
```
  private isTargetTestGame: boolean = false;
```
And then in the `setup()` function:

```
this.isTargetTestGame = settings.gameMode === GameMode.TARGET_TEST;
```
Then in `processFrame()` we can just return early if it's not a target test game.